garrysmod/lua/vgui/prop_combo.lua

```diff
@@ -10,13 +10,15 @@ function PANEL:Setup( vars )
 
 	self:Clear()
 
+	local bicon = istable( vars.icons )
```

Comment:
Why should it be there?

It's used only in loop

Reply:
I put it outside so it shold not be cheched every iteration to generate the boolean control. It is for performance.

Reply:
The question is why did you moved even further in other commit?

Reply:
Put it with the other declarations. Wherw do you perer it to be moved?